docs/faq_build.md

```diff
@@ -75,6 +75,87 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="df11", MODE:="066
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="16c0", ATTRS{idProduct}=="05df", MODE:="0666"
 ```
 
+### FreeBSD `devd` Configuration
+On FreeBSD, you'll need proper privileges to access the MCU. You can either use
+`sudo` when flashing firmware, or place these files in `/etc/devd/` and create
+a dedicated group that has permissions to write the the MCU.
+
+Create the dedicated group and add your normal user with `yourusername`:
+
+```console
+pw groupadd dfu
+pw groupmod dfu -M yourusername
+```
+
+Once added run the following to restart `devd` service:
+
+```console
+service devd restart
+```
+
+**/etc/devd/50-atmel-dfu.conf:**
+```
+# ATMEL devices ATMega32U4 (02ff4), USBKEY AT90USB1287 (0x2ffb), and ATMega32U2 (0x2ff0)
+notify 1000 {
+        match "system" "USB";
+        match "vendor" "0x03eb";
+        match "product" "0x2ff[04b]";
+        action "chown :dfu /dev/$cdev && chmod 0660 /dev/$cdev";
+};
+
+```
+
+**/etc/devd/52-tmk-keyboard.conf:**
+```
+# tmk keyboard products     https://github.com/tmk/tmk_keyboard
+notify 1000 {
+        match "system" "USB";
+        match "vendor" "0xfeed";
+        action "chown :dfu /dev/$cdev && chmod 0660 /dev/$cdev";
+};
+```
+
```

Comment:
```suggestion
```

Reply:
@fauxpark I'm happy to do so, but that certainly will conflict even *more* with the change from #8750 

Any preference on order for doing that?

Reply:
Just thinking out loud. These can probably go in first.